src/main/java/org/jabref/logic/formatter/bibtexfields/RegexFormatter.java

```diff
@@ -87,7 +90,7 @@ public String getExampleInput() {
     public static void setRegex(String rex) {
         // formatting is like ("exp1","exp2"), we want to remove (" and ")
         String rexToSet = rex;
-        rexToSet = rexToSet.substring(2, rexToSet.length() - 2);
+        rexToSet = rexToSet.substring(REMOVE_OUTER_QUOTES, rexToSet.length() - REMOVE_OUTER_QUOTES);
```

Comment:
I would replace the first `REMOVE_OUTER_QUOTES` by `"\"(".length()` and the second one by `")\"".length()`. Maybe as two variables `LENGTH_OF_QUOTE_AND_OPENING_BRACE` and `LENGTH_OF_CLOSING_BRACE_AND_QUOTE`.